content/html/encyclopedia/tables.md

```diff
@@ -0,0 +1,108 @@
+---
+Subjects:
+  - "HTML/CSS"
+  - "Web Development"
+Tags: 
+  - "Elements"
+  - "HTML & CSS"
+  - "Table"
+  - "Web Development"
+Catalog Content:  # If listing multiple URLs, please put the most relevant one first 
+  - "https://www.codecademy.com/learn/learn-html"
+---
+
+# HTML - Tables
```

Comment:
Would you mind removing the title from the Markdown and moving it to the metadata? I edited [the template](https://github.com/Codecademy/codepedia-content/blob/main/documentation/encyclopedia-entry-template.md) to reflect this, but I did it just recently, so no worries about missing the change. 
Specifically, the metadata should start with `Title: "Tables in HTML"`